lib/Conversion/TorchOnnxToTorch/DefaultDomainAtoF.cpp

```diff
@@ -601,40 +601,75 @@ void mlir::torch::onnx_c::populateDefaultDomainAtoF(
                       binder.op, resultType, operand);
                   return success();
                 });
+  // onnx.Clip with min/max as arguments introduced in opset 11
   patterns.onOp(
-      "Clip", 13, [](OpBinder binder, ConversionPatternRewriter &rewriter) {
+      "Clip", 11, [](OpBinder binder, ConversionPatternRewriter &rewriter) {
+        // https://onnx.ai/onnx/operators/onnx__Clip.html
+        // Note: version 13 added 'bfloat16' support
```

Comment:
Anything that is just superset behavior (e.g. adding type support) we can just support without comments. I wouldn't both including comments as even if a type were not support (e.g. `bfloat16`) it wouldn't change how our lowering work.s